frontend/src/components/RequestFormInputs.vue

```diff
@@ -153,6 +153,11 @@ const TOKENS: SelectorOption[] = chainsConfiguration[
 const fromChainId = ref(getChainSelectorOption(String(ethereumProvider.value.chainId.value)));
 const selectedToken = ref();
 
+const TARGET_CHAINS: Array<SelectorOption | string> = CHAINS.filter(
+  (chain: SelectorOption | string) =>
+    (chain as SelectorOption).value !== (fromChainId.value as SelectorOption).value,
```

Comment:
> I'm not sure about the typing here. First of all, you must not explicitly define `(chain: SelectorOption | string)` as TypeScript knows that itself (I tested it). Usually I like to be explicit, but here it is really not necessary and contributes to a too nested code section. It is actually pretty simple what the code should do, but it looks "complicated".

It depends on the linting options you have! I follow a more restricted linting maybe 😄 . It shows a warning on my editor and I go crazy easily because of this stuff 😛 

> Second: when `chain` can be also a `string` why are you then force cast it to a `SelectorOption`? You get `undefined == undefined` which is actually `true`. I know the `getChainSelectorOption` function returns only empty strings in the alternative case. But what if that changes? That is an assumption that is not fixed. And while being explicit, why not just unify the `if-else` cases in the referenced function and create an "empty"/default `SelectorOption`? Basically expressing that nothing has been selected. This would make the whole code more strict and straight-forward.

Maybe it's a little bit strange, but all of this hassle of adding a `string` type beside `SelectorOption` is because the only way to show a placeholder in this stupid selector component is by adding an empty string value as an option. All other values like `undefined` or `null` or even a `SelectorOption` with a `value` of empty string didn't work. I was stuck with this weird thing only to show the selector placeholder and without it, things would be easier and cleaner

Reply:
> It depends on the linting options you have! I follow a more stricted linting maybe smile . It shows a warning on my editor and I go crazy easily because of this stuff stuck_out_tongue

Interesting. I mean this is actually straight forward inference. But okay, then keep it for the moment. We have not linting pipeline anyway. 😑 

> Maybe it's a little bit strange, but all of this hassle of adding a string type beside SelectorOption is because the only way to show a placeholder in this stupid selector component is by adding an empty string value as an option. All other values like undefined or null or even a SelectorOption with a value of empty string didn't work. I was stuck with this weird thing only to show the selector placeholder and without it, things would be easier and cleaner


That is a bummer. Okay, then keep it for the moment and have it in the back of our heads that some day we might wanna find a more elegant solution here. 👍🏾